src/aap_eda/core/models/webhook.py

```diff
@@ -0,0 +1,119 @@
+#  Copyright 2024 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import uuid
+
+from django.db import models
+
+from .base import BaseOrgModel
+
+__all__ = "Webhook"
+
+EDA_WEBHOOK_CHANNEL_PREFIX = "eda_webhook_"
+
+
+class Webhook(BaseOrgModel):
+    class Meta:
+        db_table = "core_webhook"
+        indexes = [
+            models.Index(fields=["id"], name="ix_webhook_id"),
+            models.Index(fields=["name"], name="ix_webhook_name"),
```

Comment:
I'm afraid that this can be a premature optimization, since all the webhooks are going to be filtered per organization. I would suggest to remove it for now or create a composite index with organizations.

Reply:
We can do this in a separate PR